docs/src/tutorials/bseries_basics.md

```diff
@@ -0,0 +1,314 @@
+# [B-Series basics](@id tutorial-bseries-basics)
+In this tutorial we use `bseries.jl` to investigate error expansions for RK
+methods, generically or when applied to a specific ODE.
```

Comment:
```suggestion
# [B-Series basics](@id tutorial-bseries-basics)

In this tutorial we use BSeries.jl to investigate error expansions for Runge-Kutta (RK)
methods, generically or when applied to a specific ordinary differential equation (ODE).
```
Julia packages are usually written in normal font and the capitalization used in their package name (when using BSeries).
I also introduced the abbreviations since this is our first tutorial, so it might be better to be explicit.